.gitignore

```diff
@@ -1,6 +1,5 @@
+build
 *~
-Makefile
-Makefile.in
 .deps
```

Comment:
Why  should the autogenerated files not in the ignore list? We don't want them by accident versionized?

Reply:
Because the files are no longer generated. They were generated with the old autotools build scripts which was removed in 61af7d7c61c77c2663100a080392980ff646c6d3 (2016-08-30). CMake does not generate Makefile.in and the lxqt-build-tools does not allow in source tree builds (https://github.com/lxqt/lxqt-build-tools/pulls/28, https://github.com/lxqt/libfm-qt/issues/157) which would be the only way (AFAIK) to generate a Makefile at the root dir.